### DISCUSSION TITLE
Message/packet size limit with MQTT version 3.1.1

### DISCUSSION THREAD

**Discussion by Mupfelen** [mqttjs/MQTT.js#1843] [Q&A] [answered]
Hello,

i am using a device with very limited RAM. In order to avoid having to process messages that are too large, i would like to set an upper limit to the message size my application can receive. Is this possible using this library and MQTT 3.1.1? I know MQTT 5 has the capability to configure this in the CONNECT packet, is there a workaround for lower versions?

**Comment by robertsLando** [comment, marked as answer]
No one that I'm aware of unfortunately
